Dockerfile

```diff
@@ -1,26 +1,50 @@
-FROM python:3.7.9-stretch
+ARG PY_IMG_TAG=3.7.9-stretch
```

Comment:
Having the build-arg specified should allow us to do things like run tests across a range of python/node versions.